### ISSUE TITLE
Update documentation

### ISSUE THREAD

**Issue by nantonel** [kul-forbes/StructuredOptimization.jl#20]
Looks like we should update [this](https://kul-forbes.github.io/StructuredOptimization.jl/latest/solvers/)
```julia
julia> A, b = randn(10,4), randn(10);

julia> @minimize ls(A*x-b) + 0.5*norm(x);
    it |      gamma |        fpr |        tau |        FBE |
 ------|------------|------------|------------|------------|
     1 | 2.9152e-02 | 2.7656e+00 | 1.0000e+00 | 5.5181e+00 |
     9 | 2.9152e-02 | 9.9682e-05 | 1.0000e+00 | 4.4086e+00 |

julia> @minimize ls(A*x-b) st x >= 0.;
    it |      gamma |        fpr |        tau |        FBE |
 ------|------------|------------|------------|------------|
     1 | 5.8304e-02 | 1.0068e+00 | 1.0000e+00 | 6.6282e+00 |
     3 | 5.8304e-02 | 9.5210e-16 | 1.0000e+00 | 6.5654e+00 |

julia> it, slv = @minimize ls(A*x-b) st norm(x) == 2.0 with PG(maxit = 5);
    it |      gamma |        fpr |
 ------|------------|------------|
     1 | 6.1373e-02 | 2.2090e+01 |
     5 | 3.0686e-02 | 5.5190e-01 |
```

**Comment by lostella** [comment]
See #21. I would avoid putting these outputs in docstrings, exactly because of these breaking changes. Maybe we can try to make the output great again (MOGA), but providing examples without outputs is safer for now I guess.

**Comment by nantonel** [comment]
Wow you were fast!! MOGA 😆 

I think there is actually a cool feature in Documenter where the code of the example is [actually run](https://juliadocs.github.io/Documenter.jl/stable/lib/internals/expanders/#Documenter.Expanders.EvalBlocks)! However, I don't know if it works on the code appearing in `src`... 

But yep, better keep it simple!

**Comment by nantonel** [comment]
Solved in #21
